src/jaxoplanet/_src/experimental/starry/utils.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+
+from jaxoplanet._src.experimental.starry.basis import Y
```

Comment:
This import won't work (`Y` has been renamed to `Ylm` by you :D) so should be fixed or removed.